inbm/telemetry-agent/telemetry/telemetry_handling.py

```diff
@@ -303,7 +303,7 @@ def get_query_related_info(option: str, info: Dict) -> Dict:
                 del info['values'][elem]
     elif option == "version":
         info = {}
-        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()        
+        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
     elif option == "all":
```

Comment:
**<h2>$\color{#FFA500}Improvements: CodeQuality$</h2>**
**Suggestion:** Consider using a context manager for file operations to ensure that files are always properly closed even if an error occurs.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                # The existing code already uses a context manager. This suggestion is to ensure that it is used consistently throughout the codebase where file operations are performed.
```

Reply:
not constructive; the existing code is fine, as the reviewer tells us, it's just saying check the rest of the code